components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/util/SAMLSSOUtil.java

```diff
@@ -930,6 +930,11 @@ public static X509CredentialImpl getX509CredentialImplForTenant(String tenantDom
         X509CredentialImpl credentialImpl = null;
         KeyStore keyStore;
 
+        PrivilegedCarbonContext.startTenantFlow();
```

Comment:
shall we use FrameworkUtils.startTenantFlow util method

Reply:
Fixed https://github.com/wso2-extensions/identity-inbound-auth-saml/pull/303/commits/0602d43ec98400e4afb2d6971692782b5f7c3ab1